installer/sharch_body.sh

```diff
@@ -25,13 +25,21 @@ fi
 
 echo " OK."
 
+image_size=$(( $(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) / 1024))
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
 export cur_wd
 archive_path=$(realpath "$0")
 tmp_dir=$(mktemp -d)
 if [ "$(id -u)" = "0" ] ; then
     mount -t tmpfs tmpfs-installer $tmp_dir || exit 1
+    mount_size=$(df $tmp_dir | tail -1 | tr -s ' ' | cut -d' ' -f4)
+    # Extra space for tmp
+    padding=102400
```

Comment:
write the size value, also in the comment in KBs or MBs

Reply:
not adding, only checking extra space after extraction

Reply:
@antony-rheneus pls review the comments

Reply:
@antony-rheneus  Please review updated commit.